engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -268,24 +268,19 @@ class WorkflowActor(val workflowId: WorkflowId,
   }
 
   onTransition {
-    case oldState -> terminalState if terminalState.terminal =>
+    case (oldState, terminalState: WorkflowActorTerminalState) =>
       workflowLogger.info(s"transition from {} to {}. Shutting down.", arg1 = oldState, arg2 = terminalState)
       // Add the end time of the workflow in the MetadataService
       val now = OffsetDateTime.now
       val metadataEventMsg = MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.EndTime), MetadataValue(now))
       serviceRegistryActor ! PutMetadataAction(metadataEventMsg)
       terminalState match {
-        case WorkflowSucceededState =>
-          context.parent ! WorkflowSucceededResponse(workflowId)
         case WorkflowFailedState =>
           val failures = nextStateData.lastStateReached.failures.getOrElse(List.empty)
           val failureEvents = failures flatMap { r => throwableToMetadataEvents(MetadataKey(workflowId, None, s"${WorkflowMetadataKeys.Failures}[${Random.nextInt(Int.MaxValue)}]"), r) }
           serviceRegistryActor ! PutMetadataAction(failureEvents)
-
           context.parent ! WorkflowFailedResponse(workflowId, nextStateData.lastStateReached.state, failures)
```

Comment:
Because it was actually doing something useful - this carries the exception info which gets printed out to the log. 

Success was just printing out something like "hooray! a success!" 1 message prior to the general success message.

Abort wasn't doing anything at all.